### ISSUE TITLE
FoF Upload incompatibility

### ISSUE THREAD

**Issue by SKevo18** [JeromeGillard/flarum-map#9]
```
PHP Fatal error:  Declaration of JeromeGillard\FlarumMap\Templates\MapTemplate::template(): string must be compatible with FoF\Upload\Contracts\TextFormatterTemplate::template(): Illuminate\Contracts\View\View in /data/host/skeleton/vendor/jeromegillard/map/src/Templates/MapTemplate.php on line 40
```

**Comment by JeromeGillard** [comment]
@SKevo18 could you run `composer update`  and see if it brings you a solution? I've published a fix to support `flarum/core:1.8.3`.
You should see:
```
 - Upgrading jeromegillard/map (v0.1.6 => v0.1.7): Extracting archive
```

**Comment by SKevo18** [comment]
Oh wow you're fast, thank you very much! Yes, that seemed to have fixed the issue (I'll let you know if anything else shows up in the future)
